www.foia.gov/_layouts/home.html

```diff
@@ -1,19 +1,15 @@
 ---
+layout: base
 ---
-<!DOCTYPE html>
-<html>
-{% include head.html %}
-<body>
-{% include header.html %}
 <section class="usa-hero">
   <div class="usa-grid">
     <div class="usa-hero-callout usa-section-dark">
       <h2><span class="usa-hero-callout-alt">{{ page.hero.hed }}</span></h2>
       <p>{{ page.hero.text }}</p>
       <div class="hero-unit_button-group">
-        <a class="usa-button usa-button-outline" href="#agency-search">Start your request</a>
+        <a href="#learn-more">Learn more about FOIA</a>
```

Comment:
At one point @vz3 mentioned the the primary callout should be the first one, e.g. the button should be before the link.

Reply:
I don't think that's the case now -- I think we're leading with learning before submitting. This would be good to double-check with design.